### PR TITLE
Miscellaneous Fixes

### PR DESCRIPTION
**Pull request by cj81499**
Fixes a bunch of small (mostly syntax) errors.

See commit messages for more info and the comments I left on the to indicate some things I think might cause problems.

Also, note that you never actually create a `CreatePlaylist` object, so running this program wont do anything (yet). At the end of the file, on a new line with no indentation, write "`CreatePlaylist()`" to create a `CreatePlaylist` object and start your code.

### CODE REVIEW

File: create_playlist.py [hunk 1]
```diff
@@ -1,31 +1,33 @@
...
+
+
 class CreatePlaylist:
-    
+
     def _init_(self):
-        self.user_id= spotify_user_id # takes the username in our secret file 
+        self.user_id = spotify_user_id  # takes the username in our secret file
         self.spotify_token = spotify_token
         self.get_youtube_client = self.get_youtube_client()
```
Comment (cj81499) [inline create_playlist.py:27]: If you assign something to `self.get_youtube_client`, you will overwrite the `get_youtube_client` method.
Based on [line 55](https://github.com/jcraay/SpotifyPlaylist/pull/1/files#diff-bbd112e5b30074d9a647b51925de6f81R55) I think you may have meant to call this `self.youtube_client`.

File: create_playlist.py [hunk 2]
```diff
@@ -37,123 +39,129 @@ def get_youtube_client(self):
...
+
+            # save imported information
+            self.all_song_info[video_title] = {
+                "youtube_url": youtube_url,
+                "song_name": song_name,
+                "artist": artist,
+
+                # add the uri, easy to get song to put into Playlist
+                # call the function we wrote
+                "spotify_uri": self.get_spotify_uri(song_name, artist)
```
Comment (cj81499) [inline create_playlist.py:83]: The `CreatePlaylist` class has no `get_spotify_uri` method.

File: create_playlist.py [hunk 3]
```diff
@@ -37,123 +39,129 @@ def get_youtube_client(self):
...
-                        "Content-Type" : "application/json",
-                        "Authorization" : "Bearer {}".format(self.spotify_token)
-                }
-            } 
-            response_json = response.json()
-            return response.json()
+        # collect all of uris
+        uris = []
+        for song, in self.all_song_info.items():
+            uris.append(info["spotify_uri"])
```
Comment (cj81499) [inline create_playlist.py:148]: `info` is not defined. Did you mean `song`?

File: create_playlist.py [hunk 4]
```diff
@@ -1,31 +1,33 @@
...
 import googleapiclient.discovery
 import googleapiclient.errors
 import youtube_dl
- 
+
+
 class CreatePlaylist:
-    
+
     def _init_(self):
```
Comment (cj81499) [inline create_playlist.py:24]: this should be `__init__`, not `_init_` (two underscores on both sides of "init"). This type of method (and others, for example: `__add__`, `__len__` are often referred to as dunder methods (**d**ouble **under**score) or magic methods.

Comment (jcraay) [reply create_playlist.py:24]: just fixed that, Definitely never have heard of that term, Thanks for this correction!

Comment (cj81499) [reply create_playlist.py:24]: No problem! :smile:

File: create_playlist.py (hunk 2 above)
Comment (jcraay) [reply create_playlist.py:83]: I'm calling the method I wrote called "get_spotify_uri" down a couple of lines

File: create_playlist.py (hunk 4 above)
Comment (cj81499) [reply create_playlist.py:24]: just as a fyi, you *might* create merge conflicts (you and I both change a line of code, and then git doesn't know which change to accept) if you make changes to your code **before** merging this pull request.

File: create_playlist.py (hunk 2 above)
Comment (cj81499) [reply create_playlist.py:83]: one says ur**l**, the other says ur**i**

**Pull request merged**